interfaces/apparmor/backend_test.go

```diff
@@ -1267,6 +1267,27 @@ func (s *backendSuite) TestSetupHostSnapConfineApparmorForReexecWritesNew(c *C)
 	c.Check(err, IsNil)
 }
 
+func (s *backendSuite) TestSnapConfineProfileDiscardedLateSnapd(c *C) {
+	restorer := release.MockOnClassic(false)
+	defer restorer()
+	restorer = apparmor_sandbox.MockLevel(apparmor_sandbox.Full)
+	defer restorer()
+	// snapd snap at revision 222.
+	snapdInfo := snaptest.MockInfo(c, snapdYaml, &snap.SideInfo{Revision: snap.R(222)})
```

Comment:
AIUI ideall we would add a snap-confile file here  with e.g. rev 111 so that we can check that the glob is targed to the given revision and then below check that e.g. snap-confine.snapd.111 is still present.